core/tests/protractor_utils/SubTopicViewerPage.js

```diff
@@ -22,11 +22,23 @@ var action = require('./action.js');
 
 var SubTopicViewerPage = function() {
   var subTopicTileList = element.all(by.css('.protractor-test-subtopic-tile'));
+  var conceptCardList = element.all(
+    by.css('.protractor-test-concept-card-link'));
+  var conceptCardExplanation = element(
+    by.css('.protractor-test-concept-card-explanation'));
 
-  this.get = async function() {
-    var revisionTabLink = element(by.css('.protractor-test-revision-tab-link'));
-    await action.click('Revision Tab', revisionTabLink);
+  this.get = async function(subTopicName) {
     await waitFor.pageToFullyLoad();
+    var subTopicTile = element(by.cssContainingText(
+      '.protractor-test-subtopic-tile', subTopicName));
+    if (await subTopicTile.isPresent()) {
+      await action.click(subTopicName, subTopicTile);
+      await waitFor.pageToFullyLoad();
+    } else {
+      throw new Error (
+        'Sub topic ' + subTopicName + ' card is not present,'
+      );
+    }
```

Comment:
Instead of this `if present {...} else throw error` pattern, you can just use a `waitFor.presenceOf` call. That call will fail if the element isn't present, but it will also give the element some time to load in case the server is slow

Reply:
Done